src/Listener/ServiceListener.php

```diff
@@ -52,10 +45,13 @@ class ServiceListener implements ServiceListenerInterface
     protected $serviceManagers = [];
 
     /** @param null|array $configuration */
-    public function __construct(ServiceManager $serviceManager, $configuration = null)
-    {
-        $this->defaultServiceManager = $serviceManager;
-
+    public function __construct(
+        /**
+         * Default service manager used to fulfill other SMs that need to be lazy loaded
+         */
+        protected ServiceManager $defaultServiceManager,
+        $configuration = null
+    ) {
         if ($configuration !== null) {
             $this->setDefaultServiceConfig($configuration);
```

Comment:
Wait... is `$this->defaultServiceConfig` a dynamic property? :O

Reply:
it is not, it defined as `protected $defaultServiceManager` above, just different variable parameter.

Reply:
I forgot to expand the file sections in github :facepalm: 

Thanks for cross-checking!